internal/datasources/rest/handler.go

```diff
@@ -48,6 +60,27 @@ type restHandler struct {
 	// TODO implement auth
 }
 
+func initMetrics() {
+	metricsInit.Do(func() {
+		meter := otel.Meter("minder")
+		var err error
+		dataSourceRequestCounter, err = meter.Int64Counter(
+			"datasource.rest.request",
+			metric.WithDescription("Total number of data source requests issued"),
+		)
```

Comment:
If we're keeping a histogram, we don't need a separate counter -- the top bin of the histogram will give us the total counts.